.github/workflows/Release.yml

```diff
@@ -15,6 +15,6 @@ jobs:
       - name: "Build, Tag, and Push Docker Image"
         run: |
           echo "${{secrets.DOCKER_HUB_TOKEN}}" | docker login -u gopalkrishnaps --password-stdin
-          docker build -t gopalkrishnaps/intellij-android:2020.1.1-30 -t gopalkrishnaps/intellij-android:latest .
+          docker build -t gopalkrishnaps/intellij-android:2020.3.1-30 -t gopalkrishnaps/intellij-android:latest .
           docker push gopalkrishnaps/intellij-android:latest
-          docker push gopalkrishnaps/intellij-android:2020.1.1-30
+          docker push gopalkrishnaps/intellij-android:2020.3.1-30
```

Comment:
Why are you pushing it to gopals storage?

Reply:
Inspection is picking from there only no?